kolibri/core/assets/src/core-store.js

```diff
@@ -54,6 +55,9 @@ const mutations = {
   CORE_SET_LOGIN_ERROR(state, value) {
     state.core.loginError = value;
   },
+  CORE_SET_SIGN_UP_ERROR(state, value) {
```

Comment:
As this lives in the User Plugin - why is this little bit of state and a mutation in the core store?